src/pushsource/_impl/model/ami.py

```diff
@@ -99,9 +99,42 @@ class AmiPushItem(VMIPushItem):
     )
     """Billing codes associated with this image."""
 
+    release_notes = attr.ib(type=str, default=None, validator=optional_str)
+    """"Notes regarding changes in the newest version. URLs are supported.
+    
+    SPECIFICATIONS:
```

Comment:
In general, I think the docs should be consistent with other classes and fields, none of which use this kind of "SPECIFICATIONS" subheader.  The current doc style would be to write something like:

> If present, the value must not be blank and must not exceed 30,000 characters.

However, as these limits have now been documented, I wonder - would it be better to validate them instead of documenting them?  It'd be best to detect anything exceeding the limits as early as possible.

Reply:
Yes, that's a good things, I will remove specifications.